app/src/main/java/com/globant/example/mentorapp/presentation/view/fragment/ListUsersFragment.java

```diff
@@ -0,0 +1,116 @@
+package com.globant.example.mentorapp.presentation.view.fragment;
+
+import android.arch.lifecycle.LifecycleFragment;
+import android.arch.lifecycle.Observer;
+import android.arch.lifecycle.ViewModelProviders;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.DividerItemDecoration;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.globant.example.mentorapp.MentorApplication;
+import com.globant.example.mentorapp.R;
+import com.globant.example.mentorapp.data.entity.UserEntity;
+import com.globant.example.mentorapp.presentation.model.SharedUserViewModel;
+import com.globant.example.mentorapp.presentation.presenter.ListUsersPresenterImpl;
+import com.globant.example.mentorapp.presentation.view.activity.HomeScreenActivity;
+import com.globant.example.mentorapp.presentation.view.adapter.ListUsersAdapter;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+/**
+ * A placeholder fragment containing a simple view.
+ */
+public class ListUsersFragment extends LifecycleFragment implements ListUsersInterface {
+
+    public SharedUserViewModel model;
```

Comment:
The idea of MVP is to remove logic from the View and let the presenter in charge of that management.

The model should be keep in the presenter. You're view only needs a method *render* that receives the users or the error to draw.